src/vstart.sh

```diff
@@ -153,6 +153,7 @@ smallmds=0
 short=0
 ec=0
 ssh=0
+parallel=false
```

Comment:
i am wondering if we should make it "true" by default? as it's always nice to have a running cluster sooner than later.